src/hooks/opengl3.rs

```diff
@@ -184,7 +184,13 @@ impl ImguiRenderer {
         self.renderer.render(&mut self.ctx);
     }
 
-    unsafe fn cleanup(&mut self) {}
+    unsafe fn cleanup(&mut self) {
+        #[cfg(any(target_arch = "aarch64", target_arch = "x86_64"))]
+        dbg!(SetWindowLongPtrA(self.game_hwnd, GWLP_WNDPROC, self.wnd_proc as usize as isize));
+
+        #[cfg(target_arch = "x86")]
+        dbg!(SetWindowLongA(self.game_hwnd, GWLP_WNDPROC, self.wnd_proc as usize as i32));
```

Comment:
Could you please remove the `dbg!` here?

I think we _should_ probably manage the resulting errors but we should emit a well formed log entry instead. We can do it in the future, in a dedicated refactoring PR.

Reply:
Woops! Didn't mean to leave it like that, my bad 😆